enable-vcp-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.10
```

Comment:
A general comment on the location where we are placing these files. Dont we want to place it under vsphere package in kubernetes repo? This way the open source community can also maintain the scripts.